enterprise/api/v1/views.py

```diff
@@ -188,3 +195,80 @@ class EnterpriseCustomerEntitlementViewSet(EnterpriseReadOnlyModelViewSet):
     )
     filter_fields = FIELDS
     ordering_fields = FIELDS
+
+
+class EnterpriseCatalogViewSet(viewsets.ViewSet):
+    """
+    API views for `enterprise customer catalogs` api endpoint.
+    """
+    serializer_class = serializers.EnterpriseCourseCatalogReadOnlySerializer
+    permission_classes = (permissions.IsAuthenticated,)
+    authentication_classes = (OAuth2Authentication, SessionAuthentication, BearerAuthentication, JwtAuthentication)
+    throttle_classes = (ServiceUserThrottle,)
+
+    def list(self, request):
```

Comment:
@haikuginger currently, we have only one catalog per customer so this may not seem useful, but in future when we plan to have multiple catalogs for an enterprise customer then it would be necessary to have an API endpoint that returns all course catalogs for an enterprise customer.
At this time this API endpoint will return only one catalog associated with the enterprise customer.